include/ppx/command_line_parser.h

```diff
@@ -174,50 +174,70 @@ class CommandLineParser
---frame-count <N>             Shutdown the application after successfully rendering N frames. Default: 0 (infinite).
---run-time-ms <N>             Shutdown the application after N milliseconds. Default: 0 (infinite).
---gpu <index>                 Select the gpu with the given index. To determine the set of valid indices use --list-gpus.
 --headless                    Run the sample without creating windows.
---list-gpus                   Prints a list of the available GPUs on the current system with their index and exits (see --gpu).
---resolution <Width>x<Height> Specify the main window resolution in pixels. Width and Height must be two positive integers greater or equal to 1.
---screenshot-frame-number <N> Take a screenshot of frame number N and save it in PPM format.
-                              See also `--screenshot-path`.
---screenshot-path             Save the screenshot to this path. If not specified, BigWheels will create a
-                              "screenshot_frameN" file in the current working directory.
---stats-frame-window <N>      Calculate frame statistics over the last N frames only.
-                              Set to 0 to use all frames since the beginning of the application.
---use-software-renderer       Use a software renderer instead of a hardware device, if available.
-)";
+
+--list-gpus                   Prints a list of the available GPUs on the
+                              current system with their index and exits
+                              (see --gpu).
+)"
+#if defined(PPX_BUILD_XR)
+                            R"(
+--resolution <Width>x<Height> Specify the per-eye resolution in pixels. Width
+                              and Height must be two positive integers greater
+                              or equal to 1.
+)"
+#else
+                            R"(
+--resolution <Width>x<Height> Specify the main window resolution in pixels.
+                              Width and Height must be two positive integers
+                              greater or equal to 1.
+)"
 #endif
+                            R"(
+--screenshot-frame-number <N> Take a screenshot of frame number N and save it
+                              in PPM format. See also `--screenshot-path`.
+
+--screenshot-path             Save the screenshot to this destination. If not
+                              specified, BigWheels will create a
+                              "screenshot_frameN" file in the current working
+                              directory.
+
+--stats-frame-window <N>      Calculate frame statistics over the last N frames
+                              only. Set to 0 to use all frames since the
+                              beginning of the application.
+
+--use-software-renderer       Use a software renderer instead of a hardware
+                              device, if available.
+)"
+#if defined(PPX_BUILD_XR)
+                            R"(
+--xr-ui-res <Width>x<Height>  Specify the UI quad resolution in pixels.
```

Comment:
To get things to fit in a nice way with the help output. Otherwise this pushes everything further out. Another option is to select something wider than 80 columns for lines, but since that's the standard terminal output, it made sense. Yet another option was to select a different formatting, where lines wrap a bit differently.

I'm fine doing the last option, if you feel like that would be better long-term?

Reply:
I'm wouldn't be shocked to see 1 line differ a bit. Same thing happens in `grep --help`, `make --help`, `cmake --help`.
What some tools do is to add a new line:

```
short-option     some info
short-option-1   some other info
some-very-long-param
                 some other info
```

Reply:
Yep, okay did that. Changed the spacing so several of the existing arguments are now 'long', but I think it looks pretty good.